documentation/leo/03_language.md

```diff
@@ -281,6 +287,24 @@ function foo(
 }
 ```
 
+### Inline Function
+
+An inline function is declared as `inline {name}() {}`.
+Inline functions contain expressions and statements that can compute values.
+Inline functions that cannot be executed directly, 
```

Comment:
Remove stray 'that'. Also, you could consider rephrasing (or not) as proposed above for helper functions.